File: pyg_sellergoods_interface_71/src/main/java/com/pinyougou/sellergoods/service/SpecificationService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbSpecification;
import com.pinyougou.vo.Specification;
import entity.PageResult;

import java.util.List;

public interface SpecificationService {
    public PageResult<TbSpecification> findPage(int page, int size);

    public void save(Specification specification);

    public Specification findById(long id);

    public void delete(long[] ids);

    public List<TbSpecification> findAll();
}

File: pyg_manager_web_71/src/main/webapp/js/service/specificationService.js
app.service("specificationService",function ($http) {
    this.findAll = function () {
        return $http.get('../specification/findAll');
    }
    this.findPage = function (page,size) {
        return $http.get('../specification/findPage/'+page+'/'+size);
    }
    this.save = function (specification) {
        return $http.post("../specification/save", specification);
    }
    this.edit = function (id) {
        return $http.get("../specification/edit/"+id);
    }
    this.delete = function (selectIds) {
        return $http.get("../specification/delete/"+selectIds)
    }

})